packages/reporter/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@lage-run/reporter",
+  "version": "1.1.0",
+  "description": "Log reporters for Lage",
+  "repository": {
+    "url": "https://github.com/microsoft/lage"
+  },
+  "license": "MIT",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
```

Comment:
1. add `files` to this package.json. It's an include list for what gets published. Every package should have it.
2. add a README.md.
3.

Reply:
ay ay captain!